contributing/documentation/overview.rst

```diff
@@ -136,7 +136,7 @@ changes and push them:
 
     # ... do your changes
 
-    $ git push
+    $ git push origin improve_install_chapter
```

Comment:
you can ommit both, as the current branch is selected by default and the origin repo also. People have different namings, so this can cause many problems. (e.g. I call this repo origin and my fork wouterj)